tools/cldr-code/src/main/java/org/unicode/cldr/tool/GenerateMaximalLocales.java

```diff
@@ -124,7 +127,11 @@
         System.out.println("Keep containers " + cldrContainerToLanguages);
     }
 
-    private static final List<String> KEEP_TARGETS = Arrays.asList("und_Arab_PK", "und_Latn_ET");
+    private static final List<String> KEEP_TARGETS = Arrays.asList(
+        "und_Arab_PK",
+        "und_Latn_ET",
+        "hi_Latn"
```

Comment:
I think this might want to be hi_Latn_IN?

Reply:
hi_Latn is what the tool was checking for

Reply:
ok. I'd thought that the TARGETS meant the maximizations not the sources. But I haven't looked at this in ages!